docs/usage/configuration-options.md

```diff
@@ -1947,6 +1947,25 @@ Use the syntax `!/ /` like the following:
 }
 ```
 
+### matchCurrentAge
+
+Use this field if you want to match packages that lie in a certain age range.
+
+For Eg. If you want to group packages which are older than 2 months
+
+```json
+{
+  "packageRules": [
+    {
+      "matchCurrentAge": "> 2 months",
+      "groupName": "vital"
+    }
+  ]
+}
+```
+
+Invalid if used outside of a `packageRule`.
```

Comment:
```suggestion
```

We can drop this, you've set `parent = packageRules` in the code.

That should make it so this config option docs has a table row like this:

| Name | Value |
| ---- | ------ |
| parent | packageRules |